### PR TITLE
Update Sequencer Failure column

### PR DESCRIPTION
**Pull request by lucadonnoh**
note: also changes for non-rollups chains and archived projects are included

### CODE REVIEW

File: packages/config/src/layer2s/common/riskView.ts [hunk 1]
```diff
@@ -340,6 +259,86 @@ export const UPCOMING_RISK_VIEW: Layer2RiskView = makeBridgeCompatible({
...
+      ' Proposing new blocks requires creating ZK proofs.',
+  }
+}
+
+export function FORCE_VIA_L1(delay?: number): ProjectRiskViewEntry {
+  const delayString =
+    delay !== undefined ? ' for more than ' + formatSeconds(delay) : ''
+  return {
+    value: 'Force via L1',
+    description: `Users can force the sequencer to include a withdrawal transaction by submitting a request through L1. If the sequencer is down for more than ${delayString}, users can use the exit hatch to withdraw their funds.`,
```
Comment (michalsidzej) [inline packages/config/src/layer2s/common/riskView.ts:289]:
```suggestion
    description: `Users can force the sequencer to include a withdrawal transaction by submitting a request through L1. If the sequencer is down${delayString}, users can use the exit hatch to withdraw their funds.`,
```

File: packages/config/src/layer2s/common/riskView.ts [hunk 2]
```diff
@@ -340,6 +259,86 @@ export const UPCOMING_RISK_VIEW: Layer2RiskView = makeBridgeCompatible({
...
+  return {
+    value: 'Force via L1',
+    description: `Users can force the sequencer to include a withdrawal transaction by submitting a request through L1 with a ${forcedWithdrawalFeeString} fee. If the sequencer is down for more than ${delayString}, users can use the exit hatch to withdraw their funds. The sequencer can censor individual deposits, but in such case after ${maxAgeDepositUntilWithdrawableString} users can get their funds back.`,
+  }
+}
+
+export const ENQUEUE_VIA_L1: ProjectRiskViewEntry = {
+  value: 'Enqueue via L1',
+  description:
+    "Users can submit transactions to an L1 queue, but can't force them. The sequencer cannot selectively skip transactions but can stop processing the queue entirely. In other words, if the sequencer censors or is down, it is so for everyone.",
```
Comment (michalsidzej) [inline packages/config/src/layer2s/common/riskView.ts:324]:
```suggestion
    "Users can submit transactions to an L1 queue, but can't force them. The sequencer cannot selectively skip transactions but can stop processing the queue entirely. In other words, if the sequencer censors, it is so for everyone.",
```

**Review by michalsidzej — COMMENTED** [review]
Few typos

**Review by luke7211 — CHANGES_REQUESTED** [review]
zkSync Lite Escape Hatch should be white, not yellow

**Pull request merged**